test/Tensor_test.jl

```diff
@@ -141,6 +141,23 @@
         @test all(x -> ==(x...), zip(tensor, data))
     end
 
+    @testset "svd" begin
+        using LinearAlgebra
+
+        data = rand(2, 2, 2)
+        tensor = Tensor(data, (:i, :j, :k))
+        @test_throws ErrorException svd(tensor) # Throw exception if left_inds is not provided
+        @test_throws ErrorException svd(tensor, (:l,)) # Throw expcetion if left_inds ∉ labels(tensor)
+
+        U, s, Vt = svd(tensor, labels(tensor)[1:2])
+        @test labels(U)[1:2] == labels(tensor)[1:2]
+        @test labels(U)[3] == labels(s)[1]
+        @test labels(Vt)[1] == labels(s)[2]
+        @test labels(Vt)[2] == labels(tensor)[3]
+
+        @test isapprox(U * s * Vt, data)
```

Comment:
```suggestion
        U, s, V = svd(tensor, labels(tensor)[1:2])
        @test labels(U)[1:2] == labels(tensor)[1:2]
        @test labels(U)[3] == labels(s)[1]
        @test labels(V)[1] == labels(s)[2]
        @test labels(V)[2] == labels(tensor)[3]

        @test isapprox(U * s * V, data)
```

Reply:
I think that if `U * S * Vt = tensor`, then it should be `Vt` instead of `V`, as you defined in the function.

Reply:
It does not matter because we are returning `Tensor`s and `*` is an alias for `contract`. So things like transpose doesn't make much sense.